### DISCUSSION TITLE
How to determine if a backup or restore task is successful

### DISCUSSION THREAD

**Discussion by honobi** [Altinity/clickhouse-backup#996] [Q&A] [answered]
When running clickhouse-backup create_remote in the background to create a backup to S3, how can I determine if a backup task was successful? 
Similarly, when executing a restore in the background, how can I determine if the restore task was successful?

**Comment by Slach** [comment, marked as answer]
Exit code for cli usage

system.backup_actions polling for SQL interface
GET /backup/actions polling status field in REST API
